uniffi_bindgen/src/bindings/kotlin/templates/Helpers.kt

```diff
@@ -2,20 +2,44 @@
 // This would be a good candidate for isolating in its own ffi-support lib.
 
 {% if ci.iter_object_definitions().len() > 0 %}
+
+// Interface implemented by anything that can contain an object reference.
+//
+// Such types expose a `destroy()` method that must be called to cleanly
+// dispose of the contained objects. Failure to call this method may result
+// in memory leaks.
+//
+// The easiest way to ensure this method is called is to use the `.use`
+// helper method to execute a block and destroy the object at the end.
+interface Destroyable {
+    fun destroy()
+}
```

Comment:
> we should call this Disposable

This name makes sense to me; should the corresponding method be called `dispose()` for symmetry rather than `destroy()`?

Reply:
Noting for completeness, that in my local copy I have updated this to implement `AutoCloseable` for objects.